### ISSUE TITLE
[Request] Sony SKX format (MLB 2004 [PS2], Syphon Filter PS2/PSP games, etc)

### ISSUE THREAD

**Issue by modusc896d352** [vgmstream/vgmstream#1198]
i'm just leaving this here before i forget.

anyway, i came across some PS2 and PSP games from in-house Sony games that used this weird "SKX" format which is essentially a container set that consists of "bigfiles" containing streamed sounds alongside some "companion files" that hold essential information for those files so they can be played in-game.

as i uncover more and more of them that use this format, it's best to leave this issue open for now.

anyway, here are the samples.
[skx[mlb_2004][ps2].7z](https://krakenfiles.com/view/VoWv3Zujgs/file.html)
[skx[mlb_2005][ps2].7z](https://krakenfiles.com/view/amd8vQ3MFK/file.html)
[skx[mlb][psp].7z](https://krakenfiles.com/view/QdYwNvziLy/file.html)

**Comment by 0lorak** [comment]
This format -SKX- also appear in PS Vita first-party (Sony) games too.
Below you get a sample with two skx files plus it companion files.
[SKX[MLB14TS-PS Vita]](https://mega.nz/file/hRNRGaCT#rpzK1FvX_277_WqIGSikQJO9scIeB0iAo3XsytaWQig)

**Comment by modusc896d352** [comment]
yeah, i just checked the links. they're all dead. i can't really re-upload them ATM.

**Comment by 0lorak** [comment]
It is ok. When you have time look for the .at9 file header in your files.

**Comment by modusc896d352** [comment]
now that you mention it, my files didn't have anything resembling RIFF AT9 data.

for PS2 games, most SKX files had a lot of VAGp data.
for PSP games, ditto for RIFF AT3plus data.
for PS3 games, most SKX files ranged from VPK data to XVAG data, sometimes both.

that's about it it guess. your PSV files do have RIFF AT9 data and "MIDX" data though.

**Comment by modusc896d352** [comment]
just checked MLB PSV games (from MLB 12 to MLB 15) and they do indeed use SKX bigfiles.
and they 100% store RIFF AT9 data.

**Comment by 0lorak** [comment]
Because of that, those MLBTS PSV audio streams inside SKX files are relatively easy to edit.

**Comment by bnnm** [comment]
Added.
